### DISCUSSION TITLE
[Discussion]  regarding the discussion groups

### DISCUSSION THREAD

**Discussion by sorobedio** [seominjoon/kaist-ai605#132] [Q&A] [answered]
When will the discussion groups be decided and assigned papers?
 Since it will be better to properly read the discussion papers before the meeting day.

**Comment by seominjoon** [comment, marked as answer]
Hi, I just uploaded the instructions: 

https://docs.google.com/document/d/1hLzjQCiqU7zQq4RMg9A063Fv537VXmccjWk0pph2tn0/edit

**Reply by sorobedio** [reply]
Thank you you.
